src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIStructure.java

```diff
@@ -960,6 +980,88 @@ protected boolean isThereAStructureToBuild()
         return currentStructure != null;
     }
 
+    private boolean fluidDetectStep(@NotNull final StructureIterator.StructureBlock currentBlock) {
+        worker.getCitizenStatusHandler().setLatestStatus(new TranslationTextComponent("com.minecolonies.coremod.status.detecting_fluids"));
+        if (!walkToConstructionSite(currentStructure.getCurrentBlockPosition()))
```

Comment:
that needs to be properly formatted and needs javadoc, same thing for the next method.

Reply:
Javadoc: yes, i missed that.
formatting: what is wrong with the formatting? i tried to make it like everything else.

Reply:
the bracket is on the new line

Reply:
it was like that in clearStep so i kept it like that, or did you mean they should all get moved to a new line?

Reply:
all
{

}

Reply:
if that is in clear step then someone messed that one up (can you please put it on a new line for cleanStep too?)

Reply:
ok

Reply:
i hope i didn't miss any {

Reply:
awesome